src/style/content/conversation/message-list.less

```diff
@@ -723,7 +722,7 @@
 .message-asset {
   display: flex;
   align-items: flex-start;
-  margin-top: 8px;
+  margin-bottom: 6px;
```

Comment:
this replace the previous value of 24px for images and nothing for other assets.
The margin-top is now shared with all types of messages (see above)